src/DDTrace/Integrations/Memcached/MemcachedIntegration.php

```diff
@@ -362,17 +356,16 @@ private static function setServerTagsByKey($span, $memcached, $key)
     /**
      * Given a callback arg, it extract the keys depending on scalar vs associative array.
      *
-     * @param $array
-     * @return array
+     * @param mixed $keys
+     * @param string $glue
+     * @return string
      */
-    private static function extractKeys($array)
+    private static function obfuscateKeys($keys, $glue = ' ')
```

Comment:
I like that idea! Do you mind if we remove the `Util` part of the namespace and just put it under `DDTrace\Obfuscation`? I'm also wandering if we just want to namespace a function instead of a static method like, `DDTrace\toObfuscatedString()`. Thoughts? :)

Reply:
Now that I think about it, maybe a static method on a class is better so we can add a `Obfuscation::REPLACEMENT` constant and possible other obfuscation-specific things. :)

Reply:
I like to remove the `Util` part and I like to have a class with obfuscation related things 😄